### ISSUE TITLE
Chanining multiple custom commands nests all sub-steps under the first custom command only

### ISSUE THREAD

**Issue by dstrogonovs** [Shelex/cypress-allure-plugin#124]
**Describe the bug**
Chaining multiple custom Cypress commands breaks Allure step nesting. 
Note: Tested with both own custom methods in my own project and in https://github.com/Shelex/cypress-allure-plugin-example
**To Reproduce**
1. Define a custom command e.g. 
```
Cypress.Commands.add('customCommand', (one, two, ...args) => {
    cy.log('this is message from cy custom command');
    cy.log(one);
    cy.log(two);
    args.forEach((arg) => {
        cy.log(arg);
    });
});
```
2. In a spec, chain custom commands twice
3. Generate an allure report and observe the issue

**Expected behavior**
`customCommand` is logged twice and each step has all respective sub-steps nested under it
**Actual behavior**
All sub steps from BOTH custom commands are nested under the first custom command. The status of the second `customCommand` call is `Unknown`
![image](https://user-images.githubusercontent.com/91957870/153402018-b23125fb-9381-47e9-94a2-93a9ecf0809a.png)

Another example: 
Code 
```
// Command implementation
export function getByTestID(testID: string, strict = true): Cypress.Chainable {
    const locator = strict ? `[data-testid=${testID}]` : `[data-testid*=${testID}]`;
    return cy.get(locator);
}
export function clickNext(): void {
    cy.get('button').contains('Next').click();
}
// ...
Cypress.Commands.add('getByTestID', getByTestID);
Cypress.Commands.add('clickNext', clickNext);

// Usage
cy.getByTestID('product-name').type(product.productName).clickNext();
```
Results in 
![image](https://user-images.githubusercontent.com/91957870/153402139-baf0c298-666b-4d59-a2ce-97e9779eb4c4.png)

**Environment (please complete the following information):**
- Cypress version: "6.8.0" /  "9.3.1"
- OS: MacOS Big Sur 11.6

**Comment by andriivandakurov** [comment]
+1

**Comment by lashchanka** [comment]
+1

**Comment by Shelex** [comment]
Hi @dstrogonovs 
Thank you for opening an issue.

Should be resolved in v2.26.3. Could you please check it on your side?
cc: @andriivandakurov @lashchanka

**Comment by dstrogonovs** [comment]
Can confirm that it works amazingly well with version 2.26.3
![image](https://user-images.githubusercontent.com/91957870/153729888-c251e153-7704-4736-bb49-9b2d5d772b02.png)
